test/expected/agg_bookends.out

```diff
@@ -159,3 +159,209 @@ SELECT last(quantity, time) FROM btest_numeric;
  30.5
```

Comment:
I think it's better to set up these tests like the `plan_expand_hypertable_*` tests where the same tests are run under explain and also the results with and without optimizations are compared for correctness.

Reply:
I don't see the tests where results are compared. i.e. akin to `plan_expand_hypertable_results_diff`

Reply:
done